src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -1761,6 +1753,107 @@ def make_link(  # noqa: D417 (undocumented-param)
     )
 
 
+def make_audio_segment(  # noqa: D417 (undocumented-param)
```

Comment:
I would include the params because then you see the information in hover.

Reply:
I see the params on hover.